src/main/java/com/quizte/config/QuizteConfig.java

```diff
@@ -0,0 +1,44 @@
+package com.quizte.config;
+
+import lombok.AccessLevel;
+import lombok.experimental.FieldDefaults;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+import javax.sql.DataSource;
+import java.util.Map;
+
+@Configuration
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class QuizteConfig {
```

Comment:
This configuration is redundant. spring-boot-starter-data-jpa will create datasource for you. All you need to do is to set standard properties